### PR TITLE
VOA-2064: Update Welcome page

### PR DESCRIPTION
**Pull request by valydia**
# Before
Screenshot 2020-07-09 at 08 35 39

# After
Screenshot 2020-07-09 at 08 37 34

### CODE REVIEW

File: app/views/welcome.scala.html [hunk 1]
```diff
@@ -30,47 +30,24 @@
...
-
-            @govukRadios(Radios(
-                fieldset = Some(Fieldset(legend = Some(Legend(Text(messages("welcome.want.to")))))),
-                errorMessage = form.errors.asTextErrorMessageForField("submissionCategory"),
-                name = "submissionCategory",
-                items = SubmissionTypeFormProvider.options
-            ))
-
-            <p><a class="govuk-link" href=@controllers.routes.ReportStatusController.onPageLoad().url id="submissions">@messages("submission.view")</a></p>
+        <h3 class="govuk-heading-m"><a class="govuk-link" href=@controllers.routes.WelcomeController.goToStartWebFormPage().url id="create">@messages("welcome.council_tax_webform")</a></h3>
```
Comment (russellthorn) [inline app/views/welcome.scala.html:44]: this should be a <h2>. You seem to have skipped a heading level from <h1> to <h3>.

File: app/views/welcome.scala.html [hunk 2]
```diff
@@ -30,47 +30,24 @@
...
+        <p class="govuk-body">@messages("welcome.council_tax_webform.description")</p>
+    }
+    <h3 class="govuk-heading-m"><a class="govuk-link" href=@routes.WelcomeController.goToCouncilTaxStartPage().url id="councilTaxLink">@messages("welcome.council_tax_upload")</a></h3>
```
Comment (russellthorn) [inline app/views/welcome.scala.html:47]: same, heading level skipped

File: app/views/welcome.scala.html [hunk 3]
```diff
@@ -30,47 +30,24 @@
...
 
-            @govukButton(Button(attributes = Map("id" -> "submit"),
-                name = Some("submit"), content = Text(messages("site.continue"))
-            ))
-        }
-    } else {
-        <h3 class="heading-medium">
-            <a class="govuk-link" href= @routes.WelcomeController.goToCouncilTaxStartPage().url id="councilTaxLink">@messages("welcome.councilTax.url")</a>
-        </h3>
+    <h3 class="govuk-heading-m"><a class="govuk-link" href=@controllers.routes.ReportStatusController.onPageLoad().url id="submissions">@messages("submission.view")</a></h3>
```
Comment (russellthorn) [inline app/views/welcome.scala.html:50]: here 2, simply update these to `<h2>` with the same classes and nothing should change other than your heading level hierarchy wont fail accessibility.

File: conf/messages.en [hunk 4]
```diff
@@ -168,14 +169,13 @@ login.error.password.length = Password must be 100 characters or less
...
 welcome.want.to = I want to
-welcome.councilTax.url = Council Tax
-welcome.councilTax.description = Submit your Council Tax reports and view your previous Council Tax submissions.
-submissionCategory.council_tax_webform = create a Council Tax report
-submissionCategory.council_tax_xml_upload = Upload a file of Council Tax reports
-
+welcome.council_tax_webform = Create a report
+welcome.council_tax_webform.description = Text to describe the action.
+welcome.council_tax_upload = Upload a report
+welcome.council_tax_upload.description = Text to describe the action.
```
Comment (russellthorn) [inline conf/messages.en:178]: "Text to describe the action." seems like very vague content. Has this been passed by a content designer? I'm making this comment without knowing the layout and context of the page of course.

File: app/views/welcome.scala.html (hunk 1 above)
Comment (valydia) [reply app/views/welcome.scala.html]: Done in https://github.com/hmrc/voa-bar-xml-frontend/pull/138/commits/632bfb04845f593edff54f5e65e88c5dd2b5a0e8

File: app/views/welcome.scala.html (hunk 2 above)
Comment (valydia) [reply app/views/welcome.scala.html]: Done in https://github.com/hmrc/voa-bar-xml-frontend/pull/138/commits/632bfb04845f593edff54f5e65e88c5dd2b5a0e8

File: app/views/welcome.scala.html (hunk 3 above)
Comment (valydia) [reply app/views/welcome.scala.html]: Done in https://github.com/hmrc/voa-bar-xml-frontend/pull/138/commits/632bfb04845f593edff54f5e65e88c5dd2b5a0e8

File: conf/messages.en (hunk 4 above)
Comment (valydia) [reply conf/messages.en:178]: The content hasn't been confirmed by Rhys! It'll be confirmed at a letter date...
It's based on this prototype:
https://voa-bar-prototype.herokuapp.com/ - (user: voa / p/w: hmrc) - and billing authorities BA1445

Comment (russellthorn) [reply conf/messages.en:178]: ok

**Pull request merged**